R/database.R

```diff
@@ -1215,8 +1176,8 @@ get_combined_sv = function(min_vaf = 0,
 #'
 #' @param min_vaf The minimum tumour VAF for a SV to be returned.
 #' @param min_score The lowest Manta somatic score for a SV to be returned.
-#' @param pass If set to TRUE, include SVs that are annotated with PASS in FILTER column. Default is TRUE.
-#' @param pair_status Use to restrict results (if desired) to matched or unmatched results (default is to return all).
+#' @param pass If set to TRUE, include SVs that are annotated with PASS in FILTER column. Default is FALSE (since sv calls returned with get_combined_sv does not have this column).
```

Comment:
This function always includes these SVs. I think you need to say "exclude SVs that don't contain PASS" to clarify that this leads to dropping of SVs when FALSE. Right now it seems like TRUE brings in things that wouldn't otherwise be retained

Reply:
Maybe this comment needs to be revisited depending on what we decide to do with the missing FILTER column for `get_combined_sv results` (see my comment below).

Reply:
Easiest workaround is to allow PASS or NA to be kept. It would help to know whether we can interpret the combined_SV (with NA) with regard to their PASS status